umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj

```diff
@@ -815,3 +815,29 @@
   (-> collection
       (m-spec/update-version :collection "1.18.2")
       (util/update-in-all [:RelatedUrls :GetService] #(down-grade-geo-json-mime-type %))))
+
+(defmethod interface/migrate-umm-version [:collection "1.18.3" "1.18.4"]
+  [_context collection & _]
+  ;; Migrating up version 1.18.3 to 1.18.4
+  (-> collection
+      (m-spec/update-version :collection "1.18.4")))
+
+(defn- migrate-associated-doi-type-down-to-1-18-3
+  [associatedDOI]
+  (if (= "IsDescribedBy" (:Type associatedDOI))
+    (-> associatedDOI
+        (update :Type (constantly "Related Dataset"))
+        (remove-nil-keys))
+    associatedDOI))
+
+(defmethod interface/migrate-umm-version [:collection "1.18.4" "1.18.3"]
+  [_context collection & _]
+  ;; Migrating down version 1.18.4 to 1.18.3
+  ;; Remove AssociatedDOIs/Type enums: IsDescribedBy
+  (-> collection
+     (m-spec/update-version :collection "1.18.3")
+     ;; Change AssociatedDOIs/Type to 'Related Dataset' if its enum value is IsDescribedBy
+     (as-> coll (if (contains? coll :AssociatedDOIs)
+                  (-> coll
+                      (util/update-in-each [:AssociatedDOIs] migrate-associated-doi-type-down-to-1-18-3))
+                  coll))))
```

Comment:
something is up, no new line here as well as the alignment.

Reply:
Indeed.... something is wrong with my VS Code clojure lsp config, apparently. apologies